risedev.yml

```diff
@@ -70,6 +70,19 @@ risedev:
     - use: compute-node
     - use: frontend
       user-managed: true
+  
+  full:
+    - use: minio
+    - use: etcd
+    - use: meta-node
+    - use: compute-node
+    - use: frontend
+    - use: prometheus
+    - use: grafana
+    - use: zookeeper
+      persist-data: true
+    - use: kafka
```

Comment:
would you please help change the default port of Kafka to 29092 for risedev, so it's more consistent with our previous docker configuration?